### DISCUSSION TITLE
A lot of blank lines appear after exiting nvim

### DISCUSSION THREAD

**Discussion by shouyierban** [nushell/nushell#16656] [Q&A]
### Question

if I exit nvim every time after opened two files from different projects in nvim, a lot of blank lines will appear instead of continuing the previous command line.

![image](https://github.com/user-attachments/assets/7462e53c-bb6b-4392-8167-6ac4c225828b)

**Comment by sholderbach** [comment]
Can you tell us which terminal you are running with and which version of Nushell you are using?

The narrowed mention of multiple files in nvim seems oddly specific. 

In general we have a bunch of those kinds of bugs to track down where we miss estimate the prompt position (especially after resizes).

**Comment by shouyierban** [comment]
> Can you tell us which terminal you are running with and which version of Nushell you are using?
> 
> The narrowed mention of multiple files in nvim seems oddly specific.
> 
> In general we have a bunch of those kinds of bugs to track down where we miss estimate the prompt position (especially after resizes).

@sholderbach  thanks for reply, I am using nushell with wezterm in windows pc. This problem has troubled me for a long time.
